rskj-core/src/test/java/co/rsk/peg/bitcoin/UtxoTestUtils.java

```diff
@@ -0,0 +1,14 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class UtxoTestUtils {
+    public static List<Coin> coinListOf(long ... values) {
```

Comment:
Does it make sense to have a class for only this method? Could it be exported from the API of the `UtxoUtilsTest` class so we avoid creating another class?

Reply:
You make me think and reconsider it. I guess it would be better to add it to the BitcoinTestUtils class. Thanks!

Reply:
Updated!